crates/runtime/benches/bench_s3/mod.rs

```diff
@@ -28,8 +28,19 @@ pub(crate) async fn run(
     benchmark_results: &mut BenchmarkResultsBuilder,
     engine: Option<String>,
     mode: Option<Mode>,
+    bench_name: &str,
 ) -> Result<(), String> {
-    let test_queries = get_test_queries();
+    let test_queries = match bench_name {
+        "tpch" => get_test_queries(),
+        "tpcds" => {
+            if engine.clone().unwrap_or_default().as_str() == "postgres" {
```

Comment:
might be worth adding a comment here that postgres has some  query modifications to make TPC-DS pass